galvasr2/dump_cc_by_licenses.py

```diff
@@ -1,3 +1,42 @@
 #!/usr/bin/env python
 
-print("TODO")
+from align.spark.schemas import ARCHIVE_ORG_SCHEMA
+from pyspark.sql import SparkSession
+
+spark = SparkSession.builder.appName('CC-BY-license').getOrCreate()
```

Comment:
You should move this line to here:

```
if __name__ == '__main__':
    spark = SparkSession.builder.appName('CC-BY-license').getOrCreate()
    main(spark)
```

That way, if we ever need to import this file to call its main function, it won't cause a global action of allocating a new spark session.